Gemfile

```diff
@@ -3,6 +3,7 @@ git_source(:github){|repo| "https://github.com/#{repo}.git"}
 
 ruby "2.7.1"
 
+gem "bcrypt"
```

Comment:
gem này có tác dụng gì thế ô

Reply:
Đây là gem giúp người dùng có thể dễ dàng mã hóa được password lưu xuống database để tránh trường hợp nếu một ngày đẹp trời Hacker tấn công hệ thống của bạn và đánh cắp mật khẩu của user, mật khẩu của user đôi khi có thể sử dụng cho nhiều tài khoản khác nhau: tk ngân hàng, tkợ thanh toán trực tuyến, atm,... Nên việc mã hóa password của User là một bước giúp tăng security cho hệ thống của bạn (ngon) Và do cơ chế mã hóa được bcrypt sử dụng là cơ chế mã hóa một chiều nên dù Hacker có lấy được đoạn hash password thì việc tìm ra password là điều khó có thể xảy ra